assets/js/components/surveys/CurrentSurvey.js

```diff
@@ -300,15 +300,37 @@ export default function CurrentSurvey() {
 		return null;
 	}
 
+	// To properly determine the submit button label, we need to check if the current
+	// question is the last question in the survey. This is done by checking if
+	// there are any trigger conditions that reference the current question. If
+	// there are no trigger conditions, it means that the current question is the
+	// last question in the survey.
+	const isTheLastQuestion =
+		questions.some( ( { trigger_condition: conditions } ) => {
+			if ( ! Array.isArray( conditions ) || conditions.length === 0 ) {
+				return false;
+			}
+
+			for ( const condition of conditions ) {
+				if (
+					condition.question_ordinal ===
+					currentQuestion.question_ordinal
+				) {
+					return true;
+				}
+			}
+
+			return false;
```

Comment:
This can be written using `some()`, like the outer loop:

```suggestion
			return conditions.some(
				( condition ) =>
					condition.question_ordinal ===
					currentQuestion.question_ordinal
			);
```